src/Polytopes/LinearProgram.jl

```diff
@@ -126,7 +246,7 @@ end
 ### Construction of linear programs
 ###############################################################################
 ###############################################################################
-
+#=
```

Comment:
Why are you commenting out this code? If you don't need it anymore, just remove it?